main.py

```diff
@@ -1,10 +1,52 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as read_file:
+        players = json.load(read_file)
+
+    for player_name, info in players.items():
+        if not Race.objects.filter(
+                name=info["race"]["name"]).exists():
+            Race.objects.create(
+                name=info["race"]["name"],
+                description=info["race"]["description"]
+            )
+        race = Race.objects.get(
+            name=info["race"]["name"])
```

Comment:
Use can use get_or_create for this also)
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create